input/docs/cs/matlab_start.md

```diff
@@ -0,0 +1,126 @@
+title: Základní operace s obrázky
+Description: První kapitola základních operací s obrázky
+---
+>Tento dokument pracuje s [obrázkem](/zodoc/assets/img/kytka256.jpg) uloženém v proměnné`A`.
+# Načtěte a zobrazte obrázek

```

Comment:
Trochu bádám nad tím imperativem... Asi bych to dal pryč a změnil na infinitiv ("Načíst a zobrazit obrázek"), je to na vícero místech v souboru. Přidám tuto informací do konvencí..